examples/postSubmissionApplication/data/planningPermission/major.ts

```diff
@@ -0,0 +1,185 @@
+import {PostSubmissionApplication} from '../../../../types/schemas/postSubmissionApplication';
+import {planningPermissionMajorPrototype} from '../../../prototypeApplication/data/planningPermission/major';
+
+const version = process.env['VERSION'] || '@next';
+
+/**
+ * This application has been through every stage and was refused by the council and is now awaiting appeal decision
+ * No decision and no appeal = stage = assessment, status = undetermined
+ * decision and no appeal = stage = assessment, status = determined
+ * determined can refer to the council's decision or any of the appeal decisions
+ *
+ */
```

Comment:
This is a helpful pattern, there's always so many possible permutatons when it comes to examples and this is nice and explicit about what's covered :heavy_plus_sign:

Reply:
I've updated this a bit more now and have included examples of all the stages for three kinds of application that have some key differences. LDC, PP and PA. It mooches off the PrototypeApplication submissions. 😅